components/generic/ContentCard.vue

```diff
@@ -16,7 +16,7 @@
       />
       <b-card-body>
         <b-card-title>
-          {{ title }}
+          {{ title.length > 90 ? title.substring(0, 90) + $t('formatting.ellipsis') : title }}
```

Comment:
Instead of having this logic in the template, and also duplicating it in the search result component, let's make it a Vue filter named `truncate` in plugins/vue-filters.js. SearchBarPill.vue could also be made to use it.

Reply:
@rwd good suggestion, thanks. I have added the ellipsis as an argument on the truncate filter. Does that make sense to you?